leakcanary-android/src/main/java/com/squareup/leakcanary/internal/LeakCanarySingleThreadFactory.java

```diff
@@ -0,0 +1,19 @@
+package com.squareup.leakcanary.internal;
```

Comment:
I'd normally merge and edit myself, but since you're going to merge anyway, I might as well ask: can you add the copyright headers here?